api/python/tests/test_data_transfer.py

```diff
@@ -144,12 +145,14 @@ def test_list_local_url(self):
         dir_path = DATA_DIR / 'dir'
         contents = set(list(data_transfer.list_url(PhysicalKey.from_path(dir_path))))
         assert contents == set([
-            ('foo.txt', 4),
-            ('x/blah.txt', 6)
+            ('foo.txt', 3 + len(os.linesep)),
+            ('x/blah.txt', 5 + len(os.linesep))
         ])
 
     def test_etag(self):
-        assert data_transfer._calculate_etag(DATA_DIR / 'small_file.csv') == '"0bec5bf6f93c547bc9c6774acaf85e1a"'
+        small_file_tag = data_transfer._calculate_etag(DATA_DIR / 'small_file.csv')
+        # Tag will be different if the file had Windows/unix line endings
+        assert small_file_tag == ('"52a0d820c8fa7bec4f04c1325901406c"' if os.name == 'nt' else '"0bec5bf6f93c547bc9c6774acaf85e1a"')
```

Comment:
An alternative here could be to use `.gitattributes` to make sure these files are checked out with unix line endings, removing these conditional branches for the difference between how an OS checks out text files.

I went with this because it was faster than figuring out the right way to make `git` happy 😆, but if there's a reason to go with that approach instead, I can give that a try

Reply:
> An alternative here could be to use .gitattributes to make sure these files are checked out with unix line endings, removing these conditional branches for the difference between how an OS checks out text files.

I think that's the correct approach, because I think that we should treat test data, well, as data, so its newlines shouldn't be autoconverted.